### PR TITLE
FEATURE: Extend the asset model by a copyright notice

### PR DESCRIPTION
**Pull request by daniellienert**
This extends the asset model by a copyrightNotice field.
This field is automatically populated from the copyrightNotice IPTC
property when an asset is imported from an asset source.

Resolves: #2064

### CODE REVIEW

File: Neos.Media/Classes/Domain/Service/AssetSourceService.php [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <?php
+
+declare(strict_types = 1);
```
[the remark this answers is not in the document]
Comment (daniellienert) [reply Neos.Media/Classes/Domain/Service/AssetSourceService.php]: @aertmann: Thanks for asking! Following the boy scout rule, I try to add type hints and force type safe operations when possible. I discussed the strict_type declaration with @kitsunet this morning I just tried it in this PR. I will add discuss post and ask for opinions if that could get a default for new / adjusted files.

[the remark this answers is not in the document]
Comment (aertmann) [reply Neos.Media/Classes/Domain/Service/AssetSourceService.php]: okay sounds good, would it make sense to skip it for this PR to get it merged and add in a separate PR with a proper explanation for introducing it and reference to said discussion after it has been decided upon?

File: Neos.Media/Migrations/Mysql/Version20181104152203.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Migrations\AbortMigrationException;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
```
Comment (kdambekalns) [inline Neos.Media/Migrations/Mysql/Version20181104152203.php]: This is breaking 4.2.0 - there it still is `Doctrine\DBAL\Migrations\AbstractMigration`

Comment (kdambekalns) [reply Neos.Media/Migrations/Mysql/Version20181104152203.php]: I correct myself, the reason are wrong dependencies in Flow dev collection vs standalone, caused by dependabot. see https://github.com/neos/flow-development-collection/pull/1475 and https://github.com/neos/flow-development-collection/pull/1476

**Pull request merged**